docs/guide/topics/reactions.rst

```diff
@@ -0,0 +1,133 @@
+Anyone can react to a message.
+
+
+
+Adding a reaction
+~~~~~~~~~~~~~~~~~~
+
+Adding reactions is quite simple.
+
+To add a custom emoji reaction you can do the following:
+
+.. code-block:: python3
+
+    await message.add_reaction("<:amogus:819046807370989599>")
+
+Which would add a reaction to the message like so:
+
+.. image:: /images/guide/topics/reactions/add_custom_emoji.png
+    :scale: 50%
+
+
+.. note::
+
+    You must have the permission :attr:`~Permissions.read_message_history` to add reactions.
+    Additionally, if no one else has reacted to the message using this emoji, you will also need the
+    :attr:`~Permissions.add_reactions` permission.
+
+:meth:`~Message.add_reaction` can also take many types, not just :class:`str`. This includes :class:`~Emoji` or :class:`~PartialEmoji`.
+For example, the following shows an alternative way to add a custom emoji reaction:
+
+.. code-block:: python3
+
+    emoji = client.get_emoji(819046807370989599)
+    await message.add_reaction(emoji)
+
+You may notice that if you try to add a standard Discord emoji like ``:smile:``, you'll receive an error.
+This is because to add standard Discord emojis to a message, you must pass their unicode values.
+Because of this, to add 😄 / ``:smile:`` you would do either of the following:
+
+.. code-block:: python3
+
+    # pass the unicode emoji via the hexadecimal value
+    await message.add_reaction("\U0001f604")
+    
+    # or
+
+    # get the unicode emoji via the name (obtained via the unicode database)
+    await message.add_reaction("\N{SMILING FACE WITH OPEN MOUTH AND SMILING EYES}")
+
+This would then result in:
+
+.. image:: /images/guide/topics/reactions/add_standard_emoji.png
+    :scale: 50%
+
+You can find the documentation for adding reactions to a message object at :meth:`~Message.add_reaction`.
+
+Removing a reaction
+~~~~~~~~~~~~~~~~~~~~~
+
+Removing reactions is similar to adding a reaction, but you must also specify a member to remove the reaction from.
+This can be anything that meets the abc :class:`~abc.Snowflake`, like :class:`~discord.Member` or :class:`~discord.Object`.
+This means that you cannot pass simply only an id, and you must instead pass something with an ``id`` attribute.
+
+.. code-block:: python3
+
+    await message.remove_reaction(
+        "<:amogus:819046807370989599>",
+        guild.me
+    )
+
+This removes a custom emoji reaction with the id ``819046807370989599`` added by us (see :attr:`~Guild.me`) from the message:
+
+.. image:: /images/guide/topics/reactions/remove_reaction_before.png
+    :scale: 50%
+
+.. image:: /images/guide/topics/reactions/remove_reaction_after.png
+    :scale: 50%
+
+Removing all reactions of a single emoji from a message is also possible: 
+
+.. code-block:: python3
+
+    await message.clear_reaction("<:amogus:819046807370989599>")
+
+It is also possible to remove all reactions from a message, not just a singular member or emoji:
```

Comment:
This following sentence feels a touch staggered due to the previous sentence (e.g., saying "You can also x. You can also y. You can also z." with no bridges doesn't read very well).

I think it'd probably be best to fix it like:
```suggestion
Removing all reactions of a given emoji from a message is also possible: 

.. code-block:: python3

    await message.clear_reaction("<:amogus:819046807370989599>")

And we can also remove all reactions entirely:
```